src/igvfd/schemas/variant.json

```diff
@@ -0,0 +1,58 @@
+{
+    "title": "Variant",
+    "description": "Abstract schema for variants.",
+    "$id": "/profiles/variant.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "selection_criteria": {
+            "title": "Variant Selection Criteria Details",
+            "type": "array",
+            "items": {
+                "title": "The reason this variant was selected for study.",
+                "type": "string",
+                "comment": "Each lab may submit a line of text of their own selection criteria for this variant."
+            }
+        },
+        "ref": {
+            "title": "Reference Sequence",
+            "description": "The sequence found in this position in the reference.",
+            "type": "string",
+            "pattern": "^[ATCG]*$"
+        },
+        "alt": {
+            "title": "Alternative Sequence",
+            "description": "The sequence found in this position in the variant.",
+            "type": "string",
+            "pattern": "^[ATCG]*$|^-$"
```

Comment:
Sorry, but I think it is not a reference/alternative sequence but a reference/alternative allele?

Reply:
I think both work?